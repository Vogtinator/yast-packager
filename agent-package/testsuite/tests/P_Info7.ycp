{
    return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"de","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Read(.selGroups);
}
{
    return SCR::Execute(.selectSelInstall, "Minimal", true );
}
{
    return SCR::Read(.selInstallSet);
}
{
    return SCR::Read(.selDependencies);
}
{
    return SCR::Read(.selPackages);
}
{
    return SCR::Execute(.selectSelInstall, "Dmz", false );
}
{
    return SCR::Read(.selInstallSet);
}
{
    return SCR::Read(.selDependencies);
}
{
    return SCR::Read(.selPackages);
}
{
    return SCR::Execute(.deselectSelInstall, "Dmz" );
}
{
    return SCR::Read(.selInstallSet);
}
{
    return SCR::Read(.selDependencies);
}
{
    return SCR::Read(.selPackages);
}
