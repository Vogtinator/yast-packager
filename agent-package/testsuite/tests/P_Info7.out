(true)
([["Devel", " Entwicklung", "addon"], ["Games", " Spiele", "addon"], ["Gnome", " Gnome System", "addon"], ["Kde", " KDE komplett", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minimal System", "baseconf"], ["Minimal+X11", " Minimales graphisches System (ohne KDE) ", "baseconf"], ["Multimed", " Multimedia", "addon"], ["Multimedia", " Multimedia", "addon"], ["Network", " Netzwerk/Server", "addon"], ["Standard", " Standard-System", "baseconf"], ["default", " Standard-System mit Office", "baseconf"], ["xxxx", "  german xxxx", "baseconf"], ["zzz_Alles", " Alle Pakete (sehr zeitaufwendig)", "baseconf"]])
(true)
(["Minimal"])
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["aaa_base", "aaa_dir", "aaa_skel", "apmd", "ash", "at", "base", "bash", "bc", "bdflush", "bzip", "compress", "cpio", "cpp", "cracklib", "cron", "cyrus-sasl", "db", "devs", "dhcpcd", "diffutils", "e2fsprogs", "ed", "eject", "fbset", "file", "fileutils", "findutils", "finger", "freetype2", "gawk", "gdbm", "gfxboot", "glibc", "gpg", "gppshare", "grep", "groff", "gzip", "hdparm", "hwinfo", "idedma", "iputils", "isapnp", "kbd", "less", "libscr", "liby2", "libycp", "libyui", "libz", "lilo", "lsof", "lukemftp", "lvm", "m4", "mailx", "man", "mktemp", "modutils", "ncurses", "net-tools", "netcfg", "openssh", "openssl", "pam", "pam_devperm", "pciutils", "perl", "popt", "portmap", "procmail", "providers", "ps", "raidtools", "rawio", "readline", "recode", "reiserfs", "rpm", "rsh", "sash", "saxtools", "sendmail", "sh-utils", "shadow", "syslogd", "sysvinit", "tcsh", "telnet", "terminfo", "textutils", "timezone", "util-linux", "vim", "xshared", "yast", "yast2", "yast2-agent-any", "yast2-agent-fdisk", "yast2-agent-isax", "yast2-agent-modules", "yast2-agent-probe", "yast2-agent-rcconfig", "yast2-base", "yast2-config-bootfloppy", "yast2-config-bootloader", "yast2-config-environment", "yast2-config-hwinfo", "yast2-config-inet", "yast2-config-network", "yast2-config-online-update", "yast2-config-package-manager", "yast2-config-rcconfig", "yast2-config-security", "yast2-config-tune", "yast2-config-update", "yast2-config-users", "yast2-core-clients", "yast2-core-pkginfo", "yast2-core-scr", "yast2-core-wfm", "yast2-db-groups", "yast2-db-keyboard", "yast2-db-mouse", "yast2-db-timezone", "yast2-lib-sequencer", "yast2-lib-wizard", "yast2-module-support", "yast2-trans-boot", "yast2-trans-bootfloppy", "yast2-trans-hwinfo", "yast2-trans-inet", "yast2-trans-inst-bootloader", "yast2-trans-inst-environment", "yast2-trans-inst-general", "yast2-trans-inst-language", "yast2-trans-inst-packages", "yast2-trans-inst-partitioning", "yast2-trans-inst-update", "yast2-trans-inst-user", "yast2-trans-inst-x11", "yast2-trans-menu", "yast2-trans-network", "yast2-trans-online-update", "yast2-trans-package-manager", "yast2-trans-rcconfig", "yast2-trans-security", "yast2-trans-support", "yast2-trans-tune", "yast2-trans-update", "yast2-trans-users", "yast2-trans-wizard", "yast2-ui-ncurses"])
(true)
(["Dmz", "Minimal"])
($["ADD":[], "CONFLICT":[$["name":"Dmz", "selections":["Minimal"]]], "REQUIRE":[]])
(["aaa_base", "aaa_dir", "aaa_skel", "apmd", "ash", "at", "autolog", "base", "bash", "bc", "bdflush", "bind8", "bindutil", "bzip", "compress", "cpio", "cpp", "cracklib", "cron", "cyrus-sasl", "db", "devs", "dhcpcd", "diffutils", "dump", "e2fsprogs", "ed", "eject", "fbset", "file", "fileutils", "findutils", "finger", "freetype2", "gawk", "gdbm", "gfxboot", "glibc", "glibc-devel", "gpg", "gpm", "gppshare", "grep", "groff", "gzip", "hdparm", "hwinfo", "idedma", "idled", "ipchains", "iproute2", "iptraf", "iputils", "isapnp", "kbd", "less", "libgpp", "libscr", "libsmi", "liby2", "libycp", "libyui", "libz", "lilo", "lsof", "ltrace", "lukemftp", "lvm", "lynx", "m4", "mailx", "man", "mktemp", "modutils", "mt_st", "ncurses", "net-tools", "netcat", "netcfg", "openldap2-client", "openssh", "openssl", "pam", "pam_devperm", "pciutils", "pcre", "pdksh", "perl", "popt", "portmap", "postfix", "procmail", "providers", "ps", "quota", "raidtools", "rawio", "readline", "recode", "reiserfs", "rpm", "rsh", "rsync", "sash", "saxtools", "sendmail", "sh-utils", "shadow", "siga", "strace", "su1", "sudo", "syslogd", "sysvinit", "tcl", "tcpdump", "tcsh", "telnet", "terminfo", "textutils", "timezone", "util-linux", "vim", "wipe", "xntp", "xshared", "yast", "yast2", "yast2-agent-any", "yast2-agent-fdisk", "yast2-agent-isax", "yast2-agent-modules", "yast2-agent-probe", "yast2-agent-rcconfig", "yast2-base", "yast2-config-bootfloppy", "yast2-config-bootloader", "yast2-config-environment", "yast2-config-hwinfo", "yast2-config-inet", "yast2-config-network", "yast2-config-online-update", "yast2-config-package-manager", "yast2-config-rcconfig", "yast2-config-security", "yast2-config-tune", "yast2-config-update", "yast2-config-users", "yast2-core-clients", "yast2-core-pkginfo", "yast2-core-scr", "yast2-core-wfm", "yast2-db-groups", "yast2-db-keyboard", "yast2-db-mouse", "yast2-db-timezone", "yast2-lib-sequencer", "yast2-lib-wizard", "yast2-module-support", "yast2-trans-boot", "yast2-trans-bootfloppy", "yast2-trans-hwinfo", "yast2-trans-inet", "yast2-trans-inst-bootloader", "yast2-trans-inst-environment", "yast2-trans-inst-general", "yast2-trans-inst-language", "yast2-trans-inst-packages", "yast2-trans-inst-partitioning", "yast2-trans-inst-update", "yast2-trans-inst-user", "yast2-trans-inst-x11", "yast2-trans-menu", "yast2-trans-network", "yast2-trans-online-update", "yast2-trans-package-manager", "yast2-trans-rcconfig", "yast2-trans-security", "yast2-trans-support", "yast2-trans-tune", "yast2-trans-update", "yast2-trans-users", "yast2-trans-wizard", "yast2-ui-ncurses"])
(true)
(["Minimal"])
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["aaa_base", "aaa_dir", "aaa_skel", "apmd", "ash", "at", "base", "bash", "bc", "bdflush", "bzip", "compress", "cpio", "cpp", "cracklib", "cron", "cyrus-sasl", "db", "devs", "dhcpcd", "diffutils", "e2fsprogs", "ed", "eject", "fbset", "file", "fileutils", "findutils", "finger", "freetype2", "gawk", "gdbm", "gfxboot", "glibc", "gpg", "gppshare", "grep", "groff", "gzip", "hdparm", "hwinfo", "idedma", "iputils", "isapnp", "kbd", "less", "libscr", "liby2", "libycp", "libyui", "libz", "lilo", "lsof", "lukemftp", "lvm", "m4", "mailx", "man", "mktemp", "modutils", "ncurses", "net-tools", "netcfg", "openssh", "openssl", "pam", "pam_devperm", "pciutils", "perl", "popt", "portmap", "procmail", "providers", "ps", "raidtools", "rawio", "readline", "recode", "reiserfs", "rpm", "rsh", "sash", "saxtools", "sendmail", "sh-utils", "shadow", "syslogd", "sysvinit", "tcsh", "telnet", "terminfo", "textutils", "timezone", "util-linux", "vim", "xshared", "yast", "yast2", "yast2-agent-any", "yast2-agent-fdisk", "yast2-agent-isax", "yast2-agent-modules", "yast2-agent-probe", "yast2-agent-rcconfig", "yast2-base", "yast2-config-bootfloppy", "yast2-config-bootloader", "yast2-config-environment", "yast2-config-hwinfo", "yast2-config-inet", "yast2-config-network", "yast2-config-online-update", "yast2-config-package-manager", "yast2-config-rcconfig", "yast2-config-security", "yast2-config-tune", "yast2-config-update", "yast2-config-users", "yast2-core-clients", "yast2-core-pkginfo", "yast2-core-scr", "yast2-core-wfm", "yast2-db-groups", "yast2-db-keyboard", "yast2-db-mouse", "yast2-db-timezone", "yast2-lib-sequencer", "yast2-lib-wizard", "yast2-module-support", "yast2-trans-boot", "yast2-trans-bootfloppy", "yast2-trans-hwinfo", "yast2-trans-inet", "yast2-trans-inst-bootloader", "yast2-trans-inst-environment", "yast2-trans-inst-general", "yast2-trans-inst-language", "yast2-trans-inst-packages", "yast2-trans-inst-partitioning", "yast2-trans-inst-update", "yast2-trans-inst-user", "yast2-trans-inst-x11", "yast2-trans-menu", "yast2-trans-network", "yast2-trans-online-update", "yast2-trans-package-manager", "yast2-trans-rcconfig", "yast2-trans-security", "yast2-trans-support", "yast2-trans-tune", "yast2-trans-update", "yast2-trans-users", "yast2-trans-wizard", "yast2-ui-ncurses"])
